src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1445,4 +1445,7 @@ This version used in: {2}</value>
   <data name="Replace_0_with_1" xml:space="preserve">
     <value>Replace '{0}' with '{1}' </value>
   </data>
+  <data name="Add_To" xml:space="preserve">
+    <value>Add to </value>
+  </data>
```

Comment:
what is this loc string for?  why does it end with a space?

Reply:
emmm, it used to show 'add to bar'. I just realize I can put a blank in code.

Reply:
Why are we localizing ...?

Reply:
It's the name of the UI and I don't think it is meaningful. Looking for a new name

Reply:
this text doesn't sound grammatically correct.  Where does this appear in the UI?